docs/painless/painless-guide/painless-runtime-fields.asciidoc

```diff
@@ -6,7 +6,7 @@ filtering, and sorting.
 
 When defining a runtime field, you can include a Painless script that is
 evaluated at query time. This script has access to the entire context of a
-document, including the original `_source` and any mapped fields plus their
+document, including the original document at `params._source` and any mapped fields plus their
```

Comment:
```suggestion
document, including the original document <<modules-scripting-source, `_source` field>> and any mapped fields plus their
```

I would suggest linking to the page that describes how to access the _source field (with the `params._source` included in the example of that page, but also contains the `ctx._source` for update scripts), and also contains other useful notes and details.